api/src/modules/countries/country.geo.entity.ts

```diff
@@ -37,6 +37,14 @@ export class Country {
   @ApiProperty()
   @Column('geometry', { name: 'the_geom' })
   theGeom: any;
+
+   /**
+   * Bbox.
+   */
+    @ApiProperty()
+    @Column('character varying', { name: 'bbox' })
+    bbox!: number[];
```

Comment:
Most likely `Entity()` should be replaced with [ViewEntity](https://orkhan.gitbook.io/typeorm/docs/view-entities)

Reply:
No, it shouldn't do any troubles (as well as marking as @ViewColumn) - but writing explicitly will give developers some clue that it is a view and most likely one shouldn't insert into this table directly.

Reply:
Done